spec/helper/versioning_helper_spec.rb

```diff
@@ -1,3 +1,20 @@
+def mock_native_releases
```

Comment:
Hmm I think I would move this inside the `describe`, since it seems it's only used there. Probably at the end.